experimental/dds/tree2/src/feature-libraries/editable-tree/editableField.ts

```diff
@@ -142,7 +143,11 @@ export class FieldProxyTarget extends ProxyTarget<FieldAnchor> implements Editab
 			}
 		}
 
-		return cursorsFromContextualData(this.context.schema, this.fieldSchema, content);
+		return cursorsFromContextualData(
+			{ schema: this.context.schema, getFieldGenerator: defaultGetFieldGenerator },
```

Comment:
Making this.context implement this interface would be a nice way to cleanup this code a bit. Just need to add getFieldGenerator to the implementation of EditiableTreeContext. That would also be a good approach since we are going to want to make cases like this (use in editableTree) provide a generator that allocates identifiers from the session and not just provide nothing.